AlphaWallet/WalletConnect/Types/UnconfirmedTransaction+Extensions.swift

```diff
@@ -18,7 +18,7 @@ extension UnconfirmedTransaction {
         self.transactionType = transactionType
         value = transaction.value ?? BigInt("0")
         recipient = transaction.to
-        contract = .none
+        contract = transaction.to
```

Comment:
@vladyslav-iosdev this was causing the issue whereby the to value is not set and it therefore makes a new contract

Reply:
@vladyslav-iosdev could we not just remove the contract field and simply use the recipient? I see no need for the difference